DSpace/dspace-angular
# Adding themed components for EditBitstreamPageComponent and ItemStatusComponent

## Description
This PR adds theming support for the following components:
* EditBitstreamPageComponent
* ItemStatusComponent

## Instructions for Reviewers
In order to test the components:
* Modify their respective components in "custom" theme to use the themed template and add anything to those templates (e.g. a simple span with text "test") to verify its presence
* Enable the "custom" theme in config
* Start the application
* Visit the edit page of a bitstream, confirm the themed template is visible
* visit the status page of an item (edit), confirm the themed template is visible

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_
